### DISCUSSION TITLE
Resizing window while input is overflowing is deleting history

### DISCUSSION THREAD

**Discussion by PunkPun** [ghostty-org/ghostty#3789] [Feature Requests, Ideas] [answered]
You cannot scroll down to text that was consumed by resizing

https://github.com/user-attachments/assets/b4a62c0a-dd99-4756-bdb4-7b47e63c5282

**Comment by rhodes-b** [comment]
I tried to search very hard and I could not find the duplicate for this (I know it exists)

This is not a ghostty bug it happens on many other terminal emulators

**Reply by 00-kat** [reply]
#2783 perhaps?

**Reply by rhodes-b** [reply]
No its not that, that requires something spawn in the alternate buffer this does not

**Reply by rhodes-b** [reply, marked as answer]
Anyways like I said its not a ghostty bug since it reproduces in every terminal emulator I've tested in the past

Kitty

https://github.com/user-attachments/assets/c8fa46d8-d717-473b-90d8-cae1fb66a29c

Wezterm

https://github.com/user-attachments/assets/df8067da-b7f7-4c8f-a739-a0abf88dd47d

Foot

https://github.com/user-attachments/assets/2a2ae737-204b-4546-bc67-51e013f2c73b

Konsole

https://github.com/user-attachments/assets/86f73cf1-fb05-49e9-9027-20329511ff55

**Reply by PunkPun** [reply]
I moved from apples `Terminal` to Ghostty, and it doesn't have this bug

**Reply by PunkPun** [reply]
Oh my bad, it actually does. This doesn't mean it's not a bug though

**Reply by rhodes-b** [reply]
Since it happens in every single terminal emulator like i showed above it is not a ghostty bug.

It would be a shell bug if anything (im pretty sure i see it harder to reproduce in fish)

**Reply by 00-kat** [reply]
I think it might be weird prompt re-flowing edge cases which is definitely a shell issue.

Try this:
```ini
shell-integration = none
```
If that **doesn't** change anything, it's **not** a Ghostty bug.

**Reply by rhodes-b** [reply]
Shell integration doesnt matter because I see it in ghostty with my shell integration set to fish when running bash

And vise versa bash set as default and manually invoking fish

**Reply by PunkPun** [reply]
might be good to document the bug somewhere maybe? A no-fix section? Just to have a reference. Maybe people who have investigated this more indepth could give a full explanation

**Reply by 00-kat** [reply]
In my opinion this GitHub Discussion suffices.  Having a “long list of all bugs which are not ours or cannot be fixed by us” is just going to make it impossible to read due to sheer volume, and considerably increase maintenance burden (what if one of these get fixed upstream?).
